commerce-inventory-service/src/main/java/com/liferay/commerce/inventory/service/impl/CommerceInventoryWarehouseItemLocalServiceImpl.java

```diff
@@ -139,4 +177,29 @@ public CommerceInventoryWarehouseItem upsertCommerceInventoryWarehouseItem(
 				quantity);
 	}
 
+	@Override
+	public CommerceInventoryWarehouseItem
+			upsertCommerceInventoryWarehouseItemFromExternalReferenceCode(
```

Comment:
We should simplify this method name from `upsertCommerceInventoryWarehouseItemFromExternalReferenceCode` to just `upsertCommerceInventoryWarehouseItem`. The params can add clarity from there.